engine/apps/slack/client.py

```diff
@@ -87,7 +90,10 @@ def paginated_api_call_with_ratelimit(self, *args, **kwargs) -> Tuple[dict, Opti
         return cumulative_response, cursor, rate_limited
 
     def api_call(self, *args, **kwargs):
-        response = super(SlackClientWithErrorHandling, self).api_call(*args, **kwargs)
+        try:
+            response = super(SlackClientWithErrorHandling, self).api_call(*args, **kwargs)
+        except SlackApiError as err:
+            response = err.response
```

Comment:
In 1.x, a failed api call would return the error payload to you and have you handle the error. In 2.x, a failed api call will throw an exception ([migration guide docs](https://github.com/slackapi/python-slack-sdk/wiki/Migrating-to-2.x#error-handling-changes))

Reply:
even though we are no longer _directly_ calling `SlackClientWithErrorHandling.api_call`, all of the helper methods (ex. `chat_postMessage`) call `self.api_call` internally so this error handling change here will catch any newly thrown exceptions